src/routing/command.rs

```diff
@@ -25,6 +27,7 @@ use std::{
 use tokio::sync::mpsc;
 
 /// Command for node.
+#[allow(clippy::large_enum_variant)]
```

Comment:
got one removed, here adding a new one.  :)

Reply:
Right. We could either keep the `allow` or `Box` the `message` field in the `SendMessage` variant. I don't have a strong preference either way. What do you think?

Reply:
Actually, prefer to keep the `allow`, as boxing the large variant introduces lot of noise all over the codebase.

Reply:
Yeah, I'd prefer leave the `allow` as well.